### ISSUE TITLE
Chef::Exceptions::RecipeNotFound

### ISSUE THREAD

**Issue by esciara** [sous-chefs/confluence#18]
Two problems appeared recently:

```
================================================================================
Recipe Compile Error
================================================================================


Chef::Exceptions::RecipeNotFound
--------------------------------
could not find recipe server for cookbook mysql
```

and

```
================================================================================
Recipe Compile Error
================================================================================


Chef::Exceptions::RecipeNotFound
--------------------------------
could not find recipe server for cookbook mysql-chef_gem
```

This is due to the fact that
- the [v6 onwards of the mysql cookbook](https://github.com/opscode-cookbooks/mysql/blob/master/CHANGELOG.md#v600-2014-12-15) is now using LWRP and does not have recipes anymore.
- the [mysql-chef_gem cookbook](https://supermarket.chef.io/cookbooks/mysql-chef_gem), which is used in https://github.com/opscode-cookbooks/database, has had some major changes in version 1.0.0 . The [v2.3.1 of the database cookbook](https://github.com/opscode-cookbooks/database/blob/master/CHANGELOG.md#v231-2014-12-13) fixes this. However, I had to delete the `Berksfile.lock` file before running a `berks install` so that v2.3.1 is actually installed.

So to summarise, solution is to:
- lock the cookbook versions in `metadata.rb`:

``` ruby
%w(apache2 mysql_connector postgresql).each do |cb|
  depends cb
end

depends 'database', '~> 2.3'
depends 'mysql', '~> 5.0' 
```
- delete the `Berksfile.lock`
- run `berks install`

**Comment by MuscleRush** [comment]
Hi

i m very new to Chef   and really all i want to do is deploy mysql cookbook and i m getting similar error
that you outlined. 

 " could not find recipe default for cookbook mysql "

i m also getting 

'Recipe Compile Error "

i m reading this and i have no idea how i should fix it.. any help is appreciated. 

thank you

**Comment by legal90** [comment]
Fixed
